src/com/yetanalytics/lrs_admin_ui/views/footer.cljs

```diff
@@ -1,8 +1,21 @@
-(ns com.yetanalytics.lrs-admin-ui.views.footer)
+(ns com.yetanalytics.lrs-admin-ui.views.footer
+  (:require [goog.string :refer [format unescapeEntities]]
+            goog.string.format))
 
 (defn footer []
   [:footer {:class "home-footer"}
    [:div {:class "footer-wrapper"}
-    [:span "@Copyright 2021"]"   |  "
-    [:a {:class "text-white", :href "#"} "Terms and Conditions"]"   |  "
-    [:a {:class "text-white", :href "#"} "Privacy Policy"]]])
+    [:span {:class "text-white"}
+     (format "%s2021 " (unescapeEntities "&copy;"))
```

Comment:
Maybe put a comment here so that later you won't be like "what the heck does this do?"

Reply:
done.

Reply:
Oh I expecting that you were going to point out that this is a copyright symbol. It could be made more obvious, but I guess you'll see the year text and put two and two together.

Reply:
responded in slack